.bingo/gomplate.mod

```diff
@@ -0,0 +1,5 @@
+module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
+
+go 1.14
+
+require github.com/hairyhenderson/gomplate/v3 v3.8.0 // cmd/gomplate
```

Comment:
Using Gomplate to create the install script for the different release artifacts.